components/director/internal/tenantfetcher/README.md

```diff
@@ -0,0 +1,103 @@
+    }
+  ],
+  "totalResults": 27,
+  "totalPages": 1
+}
+```
+
+The **eventData** field contains an escaped JSON string with the following fields that you can configure using values overrides:
+- `$id` - specifies a unique tenant ID
+- `$name` - specifies the tenant name
+- `$discriminator` - specifies an optional field that can be used to distinguish different types of tenants
+
+#### Tenant deletion endpoint
+
+The endpoint should return the following JSON payload:
+```json
+{
+  "events": [
+    {
+      "eventData": "{\"$id\":\"837d023b-782d-4a97-9d38-fecab47c296a\",\"$name\":\"Tenant 1\"}"
+    }
+  ],
+  "totalResults": 27,
+  "totalPages": 1
+}
+```
+
+The **eventData** field contains an escaped JSON string with the following fields that you can configure using values overrides:
+- `$id` - specifies a unique tenant ID
+- `$name` - specifies the tenant name
+
+#### Tenant update endpoint
+
+The endpoint should return the following JSON payload:
+```json
+{
+  "events": [
+    {
+      "eventData": "{\"$id\":\"837d023b-782d-4a97-9d38-fecab47c296a\",\"$name\":\"Tenant 1\"}"
+    }
+  ],
+  "totalResults": 27,
+  "totalPages": 1
+}
+```
+
+The **eventData** field contains an escaped JSON string with the following fields that you can configure using values overrides:
+- `$id` - specifies a unique tenant ID
+- `$name` - specifies the tenant name
+
+## Configuration
+
+Tenant Fetcher binary allows you to override some configuration parameters. You can specify the following environment variables:
+
+| Parameter | Description |  Default value |
+|-----------|-------------|---------------|
+| **global.tenantFetcher.enabled** | Parameter that enables the Tenant Fetcher CronJob | `false` |
+| **global.tenantFetcher.providerName** | Name of the tenants provider | `"compass"` |
+| **global.tenantFetcher.schedule** | Parameter that specifies how often Tenant Fetcher fetches information about tenants | `"*/5 * * * *"` |
+| **global.tenantFetcher.oauth.client** | OAuth 2.0 client ID | None |
+| **global.tenantFetcher.oauth.secret** | OAuth 2.0 client Secret | None |
+| **global.tenantFetcher.oauth.tokenURL** | Endpoint for fetching an OAuth 2.0 access token to the Tenant Events API | None |
+| **global.tenantFetcher.endpoints.tenantCreated** | Tenant Events API endpoint for fetching created tenants | `"127.0.0.1/events?type=created"` |
+| **global.tenantFetcher.endpoints.tenantDeleted** | Tenant Events API endpoint for fetching deleted tenants | `"127.0.0.1/events?type=deleted"` |
+| **global.tenantFetcher.endpoints.tenantUpdated** | Tenant Events API endpoint for fetching updated tenants | `"127.0.0.1/events?type=updated"` |
+| **global.tenantFetcher.fieldMapping.idField** | Name of the field in the event data payload containing the tenant name | `"id"` |
+| **global.tenantFetcher.fieldMapping.nameField** | Name of the field in the event data payload containing the tenant ID | `"name"` |
```

Comment:
Aren't these two mixed up..? Look at the parameter names, their descriptions, and default values. I think the descriptions are the other way round.